.mergify/config.yml

```diff
@@ -0,0 +1,49 @@
+pull_request_rules:
+  - name: automatic backport to all supported distribution
+    description: |
+      This rule backports a PR to all supported distribution branches when it is merged.
+    conditions:
+      - base=rolling
+      - merged
+    actions:
+      backport:
+        branches:
+          - jazzy
+          - humble
```

Comment:
most likely it should be backported to all distros, we do not really expect any API dependent changes for each distro. so without having any labels, we want it to create backports to all distros.